sample/src/main/java/com/fueled/flowr/sample/MainActivity.java

```diff
@@ -35,7 +35,8 @@ protected void onCreate(Bundle savedInstanceState) {
         setSupportActionBar(binding.toolbar);
 
         if (getFlowr().getCurrentFragment() == null) {
-            getFlowr().open(HomeFragment.class)
+            getFlowr()
+                    .open(HomeFragment.class)
```

Comment:
@julien-fueled I think we can change this to `getFlowr().open(getIntent(), HomeFragment.class)` and remove the `setDeepLinkContent()` below, so if the passed intent has deep link data it can be used to open the next fragment otherwise `HomeFragment` would be displayed.